api/serializers/shared.py

```diff
@@ -0,0 +1,27 @@
+from typing import Any
+
+from rest_framework import serializers
+
+from api.exceptions import ConflictError
+from api.models import Group
+
+
+class PersonTransferSerializer(serializers.Serializer[Any]):
+    to_group_id = serializers.IntegerField()
+    from_group_id = serializers.IntegerField()
+
+    def validate(self, attrs: dict[str, Any]) -> dict[str, Any]:
+        try:
+            to_group = Group.objects.get(pk=int(attrs["to_group_id"]))
+        except Group.DoesNotExist:
+            raise ConflictError(f"Group {attrs['to_gorup_id']} is no found")
```

Comment:
```suggestion
            raise ConflictError(f"Group {attrs['to_group_id']} not found")
```
I spotted this typo in `StudentTransferSerializer`, but just corrected it before merging. Not sure it's a big deal that no tests broke (should have thrown `KeyError`)

Reply:
`to_group` is always exists in tests so it is good that you catched it at this stage